systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -126,9 +129,18 @@ void testDeployUndeploy(ExtensionContext extensionContext) {
                 "offset.storage.topic=" + KafkaConnectResources.configStorageTopicOffsets(testStorage.getClusterName()) + "\n");
 
         resourceManager.createResourceWithWait(extensionContext, KafkaTemplates.kafkaEphemeral(testStorage.getClusterName(), 3).build());
-        resourceManager.createResourceWithWait(extensionContext, KafkaConnectTemplates.kafkaConnect(testStorage.getClusterName(), testStorage.getNamespaceName(), 1).build());
+        resourceManager.createResourceWithWait(extensionContext, KafkaConnectTemplates.kafkaConnect(testStorage.getClusterName(), testStorage.getNamespaceName(), connectReplicasCount).build());
+
+        LOGGER.info("Connect cluster my-cluster deployed OK");
```

Comment:
I guess this is not needed to log, as it is already stated inside the create method or? (I know it was already in place when you touched the code)